src/nurse/tests/test_models.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime, timedelta
+
+import pytest
+
+from nurse.models import Prescription
+
+
+@pytest.mark.django_db
+class TestPrescriptionManager:
+    def setup_method(self):
+        Prescription.objects.all().delete()
+
+        Prescription.objects.create(
+            prescribing_doctor="Dr A",
+            start_date=datetime.now().date() - timedelta(days=3),
+            end_date=datetime.now().date() + timedelta(days=4),
+        )
+
+        Prescription.objects.create(
+            prescribing_doctor="Dr B",
+            start_date=datetime.now().date() - timedelta(days=10),
+            end_date=datetime.now().date() + timedelta(days=10),
+        )
+
+    def test_prescriptions_expiring_in_next_seven_days(self):
+        prescriptions = Prescription.objects.expiring_soon()
+        assert prescriptions.count() == 1
+        assert prescriptions.first().prescribing_doctor == "Dr A"
+
+    def test_prescriptions_expiring_in_next_eleven_days(self):
+        prescriptions = Prescription.objects.expiring_soon(days=11)
+        assert prescriptions.count() == 2
+
+    def test_prescription_expiring_today(self):
+        Prescription.objects.create(
+            prescribing_doctor="Dr C",
+            start_date=datetime.now().date() - timedelta(days=5),
+            end_date=datetime.now().date(),
+        )
+        prescriptions = Prescription.objects.expiring_soon(days=0)
+        assert prescriptions.count() == 1
+        assert prescriptions.first().prescribing_doctor == "Dr C"
+
+    def test_prescription_started_today(self):
+        Prescription.objects.all().delete()
```

Comment:
The fixture is already doing so. Plus each test is done in a transaction which is rolled back after so DB should already be void